app/views/base/_form_progress.html.erb

```diff
@@ -22,6 +22,16 @@
       <%= f.submit "Save and Duplicate Metadata", class: 'btn btn-primary save' %>
 
     <% end %>
+
+    <% if f.object.resource.is_a?(Numismatics::Issue) && params[:controller] == "numismatics/issues" %>
```

Comment:
I'm not certain precisely how this should be refactored - perhaps a Class method on `EphemeraFolder`, `Numismatics::Issue`, and `Numismatics::Coin`?